js/witness_calculator.js

```diff
@@ -111,6 +133,15 @@ export default async function builder(code, options) {
 
         return String.fromCharCode.apply(null, bytes);
     }
+
+    function printSharedRWMemory () {
+        const shared_rw_memory_size = instance.exports.getFieldNumLen32();
+        const arr = new Uint32Array(shared_rw_memory_size);
+        for (let j=0; j<shared_rw_memory_size; j++) {
+            arr[shared_rw_memory_size-1-j] = instance.exports.readSharedRWMemory(j);
```

Comment:
Does circom1 export this?

Reply:
I don't think so, we're breaking compatibility with circom 1 elsewhere as well.